CONTRIBUTING.md

```diff
@@ -2,10 +2,33 @@
 
 First of all, thanks for the decision to contribute to DHIS2! :heart:
 
+To contribute to any of the DHIS2 repositories, please follow a few guidelines.
+
+- Follow Code Standards
```

Comment:
I don't think `Code Standards` should be in title-case. All other points using sentence-case, which makes sense here too.